lib/govuk_ab_testing/ab_test.rb

```diff
@@ -0,0 +1,23 @@
+module GovukAbTesting
+  class AbTest
+    attr_reader :experiment_name
+
+    def initialize(experiment_name)
+      @experiment_name = experiment_name
+    end
+
+    # Internal name of the header
+    def request_header
+      "HTTP_GOVUK_ABTEST_#{experiment_name.upcase}"
+    end
+
+    def response_header
+      "GOVUK-ABTest-#{cookie_name}"
+    end
+
+    # `example` -> `Example`
+    def cookie_name
+      experiment_name.capitalize
```

Comment:
Hmm, I think I just confused myself in that last comment, sorry. We do need the `cookie_name` method, but it should return `"ABTest-@ab_test_name"`.